package.json

```diff
@@ -1,25 +1,19 @@
 {
   "version": "0.0.1",
   "devDependencies": {
-    "husky": "^1.1.4",
     "lerna": "^2.5.1",
-    "lint-staged": "^8.0.0",
-    "prettier": "^1.15.2"
+    "npm-run-all": "^4.1.5"
   },
-  "precommit": "lerna run --concurrency 1 --stream precommit",
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged"
-    }
-  },
-  "lint-staged": {
-    "*.{js,json,css,md}": [
-      "prettier --write",
-      "git add"
-    ]
+  "engines": {
+    "node": "^8.12.0 || >=9.7.0"
```

Comment:
I'm new here, but any reason we're using node 8 instead of newer like 12? 12 should be much faster than 8 and I don't see any dependencies that would make us stuck on 8.

Reply:
IIRC node v8.10 is the last supported version (like in CRA https://create-react-app.dev/docs/getting-started#creating-an-app), so it's more a matter of common agreement 😄 
And you can still use 12 because of the `>=9.7.0`

Reply:
Gotcha! thank you :)